akka-http-core/src/main/scala/akka/http/scaladsl/model/headers/HttpChallenge.scala

```diff
@@ -24,6 +25,8 @@ final case class HttpChallenge(scheme: String, realm: String,
 
 object HttpChallenges {
 
+  def negotiate(): HttpChallenge = HttpChallenge("Negotiate", "")
+
   def basic(realm: String): HttpChallenge = HttpChallenge("Basic", realm)
 
   def oAuth2(realm: String): HttpChallenge = HttpChallenge("Bearer", realm)
```

Comment:
@2beaucoup I agree, they do seem a bit superfluous, but they did get merged somewhere along the line? I'm happy to pull them all out